frcxx_nft/src/state.rs

```diff
@@ -1,4 +1,6 @@
 //! Abstraction of the on-chain state related to NFT accounting
+use std::collections::HashSet;
```

Comment:
You might already have this worked out but how is the cbor serialization of this type defined and what tradeoffs are motivating using this?  It's showing up on chain below and there's not a lot of precedent for having map datastructures serialized inline in state.  I am suspicious this is going to be strictly worse, in terms of storage overhead and gas cost of operations, than other datatypes more commonly used.

My guess is you either want a HAMT, an inline cbor array, or an inline map type with a more specialized serialization than what you get by sending the standard hash set through cbor tuple encoding.

Reply:
I suspect you are right. The reference impl will attempt to target the majority use case where I expect the operators list to be comparatively small. So for now, [I've gone with a Vec<ActorID>](https://github.com/helix-onchain/filecoin/pull/130/commits/372b8995bc0f270b467962fd390a500f61d692bf) with order maintained in-memory when operators are added and removed

I'll benchmark this properly against some alternatives when the library is finished to compare options for optimizing gas vs storage etc.